.changeset/silent-files-know.md

```diff
@@ -0,0 +1,18 @@
+---
+'@vanilla-extract/babel-plugin-debug-ids': minor
+'@vanilla-extract/css': minor
+'@vanilla-extract/dynamic': minor
+'@vanilla-extract/esbuild-plugin': minor
+'@vanilla-extract/integration': minor
+'@vanilla-extract/jest-transform': minor
+'@vanilla-extract/next-plugin': minor
+'@vanilla-extract/private': minor
+'@vanilla-extract/recipes': minor
+'@vanilla-extract/rollup-plugin': minor
+'@vanilla-extract/sprinkles': minor
+'@vanilla-extract/css-utils': minor
+'@vanilla-extract/vite-plugin': minor
+'@vanilla-extract/webpack-plugin': minor
```

Comment:
Lets just release `css` and `dynamic` as `patch` as they're the ones with consumer facing changes.

Reply:
Do you mean this?
```md
---
'@vanilla-extract/css': patch
'@vanilla-extract/dynamic': patch
---
```

`css-utils` and `sprinkles` are also affected. The example in the PR specifically mentions `sprinkles`.

Reply:
Cool, add those too than. 👍